node/src/components/consensus/cl_context.rs

```diff
@@ -0,0 +1,63 @@
+use std::rc::Rc;
+
+use tracing::info;
+
+use crate::{
+    components::consensus::{
+        candidate_block::CandidateBlock,
+        traits::{Context, ValidatorSecret},
+    },
+    crypto::{
+        asymmetric_key::{self, PublicKey, SecretKey, Signature},
+        hash::{self, Digest},
+    },
+    types::CryptoRngCore,
+};
+
+pub(crate) struct ClSecret {
```

Comment:
What do you think about calling it just `Secret`? By putting it in the `cl_context` module it would have the almost-fully qualified name as `cl_context::Secret`.

Reply:
Or `Keypair`?

Reply:
Sure, even better.